jsx/FilterableDataTable.js

```diff
@@ -17,7 +18,7 @@ import Filter from 'jsx/Filter';
  *
  * Deprecates Filter Form.
  */
-class FilterableDataTable extends Component {
+class FilterableDataTable extends PureComponent {
```

Comment:
not in **24.1**

_although purecomponents seem interesting from a performance stand-point, I'm not sure if this is necessary here and can likely be removed so as to not cause trouble_

to be tested, although basic assumption is to adopt **24.1**